lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -688,9 +688,9 @@ def run(scope, run_opts = nil)
       if @run_opts[:dump_only]
         dump_commands
       else
-        unless @configured
+        unless @run_opts[:configured]
           run_config_commands
-          @configured = true
+          @run_opts[:configured] = true
```

Comment:
The changes in `qemu_machine.rb` break the scenario where the `run` method is called multiple times without `configured` specified.